packages/vue/src/components/molecules/SfGallery/SfGallery.js

```diff
@@ -40,6 +45,18 @@ export default {
     };
   },
 
+  computed: {
+    mapPictures() {
```

Comment:
I'm thinking about product images on 3 sizes: `{ small, normal, big }`
- `small` is intended to be used for thumbs, but may also be used as picture source for product card on small devices;
- `normal` is intended to be the default for product card (and maybe others);
- `big` is used on gallery stage;

Then `SfImage` is using `small/normal` on picture tag because it's the case for product card, but for gallery it should use `normal/big`, so in this case I have to map the objects before passing to sfimage.

Maybe we can edit `SfImage` api to receive src as String/Array instead of String/Object, or we can add another prop, like `mobileSrc`, wdyt?

Reply:
@leomp12 now I understand why you use mapPicture() and now for me is ok